src/routes/user/validation.js

```diff
@@ -0,0 +1,21 @@
+const Joi = require('@hapi/joi');
+
+export const registerValidation = (data) => {
+  const schemaUser = Joi.object({
+    name: Joi.string().pattern(new RegExp('^[A-Za-z0-9]{6,255}$')).required(),
+    email: Joi.string().email().required(),
+    password: Joi.string().min(6).required(),
+    phoneNr: Joi.number().min(9).required(),
+  });
+
+  return schemaUser.validate(data);
+};
+
+export const loginValidation = (data) => {
+  const schemaUser = Joi.object({
```

Comment:
As this validation schema is a subset of `registerValidation`, maybe `registerValidation` could just be extending this validation schema to not declare the validations for `email` and `password` twice?